content/v1.12/concepts/managed-resources.md

```diff
@@ -3,630 +3,745 @@ title: Managed Resources
-`resourceGroupNameRef` is filled with the name of that `ResourceGroup` managed
-resource. Then in the last step, Crossplane fills the actual `resourceGroupName`
-field with whatever format Azure accepts it. Once a dependency is resolved, the
-controller never changes it.
-
-Users are able to specify any of these three fields:
+### Deleting
+`Reason: Deleting` indicates the Provider is attempting to delete the managed
+resource. 
 
-- Selector to select via labels
-- Reference to point to a determined managed resource
-- Actual value that will be submitted to the provider
+```yaml
+Conditions:
+  Type:                  Ready
+  Status:                False
+  Reason:                Deleting
+```
 
-It's important to note that in case a reference exists, the managed resource
-does not create the external resource until the referenced object is ready. In
-this example, creation call of Azure MySQL Server will not be made until
-referenced `ResourceGroup` has its `status.condition` named `Ready` to be true.
 
-## Management Policies
+### Paused
+`Reason: ReconcilePaused` indicates the managed resource has a [Pause](#paused)
+annotation 
+```yaml
+Conditions:
+  Type:                  Synced
+  Status:                False
+  Reason:                ReconcilePaused
+```
 
-Crossplane offers a set of management policies that allow you to define the
-level of control it has over external resources. You can configure these
-policies using the `spec.managementPolicy` field in the managed resource
-definition. The available policies include:
+<!-- vale off -->
+### ReconcileError
+<!-- vale on -->
+`Reason: ReconcileError` indicates Crossplane encountered an error while
+reconciling the managed resource. The `Message:` value of the `Condition` helps
+identify the Crossplane error. 
 
-- `FullControl (Default)`: With this policy, Crossplane fully manages and
-controls the external resource.
-- `ObserveOnly`: With the ObserveOnly policy, Crossplane only observes the
-external resource without making any changes or deletions.
+```yaml
+Conditions:
+  Type:                  Synced
+  Status:                False
+  Reason:                ReconcileError
+```
 
-{{<hint "important" >}}
-Management policies are an experimental feature, and the API is
-subject to change.
-{{< /hint >}}
 
-To use management policies, you must enable them with the
-`--enable-management-policies` flag when starting the provider controller.
 
-## Importing Existing Resources
 
-If you have some resources that are already provisioned in the cloud provider,
-you can import them as managed resources and let Crossplane manage them. What
-you need to do is to enter the name of the external resource as well as the
-required fields on the managed resource. For example, let's say I have a GCP
-Network provisioned from GCP console and I would like to migrate it to
-Crossplane. Here is the YAML that I need to create:
+### Success
```

Comment:
should this heading be `ReconcileSuccess` to match the reason the section is about? we do the same for `ReconcileError` just above